app/views/publishers/home.html.slim

```diff
@@ -86,31 +87,27 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
                       =t("promo.dashboard.confirmed").upcase
                     .promo-panel-number
                       = publisher_referral_totals[PromoRegistration::FINALIZED]
-                .promo-flex-row.promo-flex-row-sub
-                  =t("promo.dashboard.check_statement")
             - else
               .promo-panel-item.promo-panel-item-alert
                 #promo-panel-alert
                   = t("promo.dashboard.add_channel")
       - elsif current_publisher.channels.empty?
-        .col
+        .col.bot-marg
```

Comment:
Let's not use ".bot-marg", instead let's use the spacing utilities defined by bootstrap itself 
https://getbootstrap.com/docs/4.0/utilities/spacing/

`mb-1` or `mb-2`